components/ui/timeline.tsx

```diff
@@ -0,0 +1,42 @@
+type TimelineProps = {
+  active: string;
+};
+
+export function Timeline({ active }: TimelineProps) {
+  return (
+    <div className="flex flex-row items-center justify-center mt-2">
+      <p
+        className={
+          `text-sm border-2 px-2 rounded-full ` +
+          (active === "experiment"
+            ? "border-blue-300 text-blue-500"
+            : "border-grey text-slate-500")
```

Comment:
I suggest pulling the ternary expression out into a Javascript variable assignment before the return. This would make the code easier to skim by avoiding having to switch levels of abstraction in the middle of a line.